scripts/build/aflgo-build.sh

```diff
@@ -41,18 +41,18 @@ cmake -G "Ninja" \
       -DLLVM_USE_SANITIZER=Memory -DCMAKE_INSTALL_PREFIX=/usr/msan/ \
       -DLIBCXX_ENABLE_SHARED=OFF -DLIBCXX_ENABLE_STATIC_ABI_LIBRARY=ON \
       -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86" \
-       ~/build/llvm_tools/llvm-4.0.0.src
+       ~/build/llvm_tools/llvm-11.0.0.src
 ninja cxx; ninja install-cxx
 
 # Install LLVMgold in bfd-plugins
-mkdir /usr/lib/bfd-plugins
+mkdir -p /usr/lib/bfd-plugins
 cp /usr/local/lib/libLTO.so /usr/lib/bfd-plugins
 cp /usr/local/lib/LLVMgold.so /usr/lib/bfd-plugins
 
 # install some packages
 export LC_ALL=C
 apt-get update
-apt install -y python-dev python3 python3-dev python3-pip autoconf automake libtool-bin python-bs4 libclang-4.0-dev
+apt install -y python-dev python3 python3-dev python3-pip autoconf automake libtool-bin python-bs4 # libclang-11.0-dev
 python3 -m pip install --upgrade pip
 python3 -m pip install networkx pydot pydotplus
```

Comment:
Can you also add the build dependencies for distance_calculator here and directly build here, too?